packages/editor/src/lib/editor/Editor.ts

```diff
@@ -5283,12 +5296,15 @@ export class Editor extends EventEmitter<TLEventMap> {
 				}
 			}
 
+			if (!distance) distance = 0
```

Comment:
what's this about? as far as i can tell, every branch of the if thing above sets distance. even then, shouldn't this be `Infinity` or something to indicate a miss? At the very least this could use a comment